src/javascript/ContentManager/ContentManager.utils.test.js

```diff
@@ -0,0 +1,27 @@
+import {removeFileExtension, getNewCounter} from './ContentManager.utils';
+
+describe('zip utils', () => {
+    it('should remove file extension', () => {
+        expect(removeFileExtension('file.jpg')).toEqual('file');
+        expect(removeFileExtension('file')).toEqual('file');
+        expect(removeFileExtension('file.txt')).toEqual('file');
```

Comment:
I don't think this last expectation is adding much value since it's very similar to the first one.
May be valuable to test the following filenames:
- one containing several dots
- one ending with a dot